scripts/upgrade

```diff
@@ -205,35 +99,48 @@ ynh_add_nginx_config
 # VERSION TO THE NEXT ONE
 #=================================================
 
+current_version=$(grep OC_VersionString "$install_dir/version.php" | cut -d\' -f2)
+current_major_version=${current_version%%.*}
+
 # Define a function to execute commands with `occ`
 exec_occ() {
     # Backward compatibility to upgrade from older versions
-    if [ $current_major_version = "last" ] || [ $current_major_version -ge 24 ]
+    if [ $current_major_version = "last" ] || [ $current_major_version -ge 26 ]
+    then
+        NEXTCLOUD_PHP_VERSION="8.2"
```

Comment:
Just to avoid problems during future migrations, i'd use `YNH_PHP_VERSION` here instead
```suggestion
        NEXTCLOUD_PHP_VERSION="$YNH_PHP_VERSION"
```

Reply:
That's a bad idea IMO as `$YNH_PHP_VERSION` comes from packaging v1 and is hardcoded to `7.4`

Reply:
Heuarg ineed it's not in _common.sh anymore so it defaults as 7.4 on bullseye ... Didnt even think that was the case, but anyway I was paranoiac about the fact that calling ynh_install_app_dependencies may indirectly change that value .. maybe it doesnt ...

I was thinking that maybe we can be more robust by deducing the value from the manifest or something, but in any case there's no "no maintenance magic" solution here : next time we bump the php version used, we need to define a new "if" case